src/SearchResults.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+import moment from "moment";
+import Bookings from "./Bookings";
+import "./SearchResults.css";
+
+function SearchResults(props) {
+  const [isSelected, setSelection] = useState(false);
+
+  const changeColors = () => {
+    setSelection(!isSelected);
+  };
+
+  return (
+    <div>
+      <table className="table">
+        <thead className="thead-dark">
+          <tr>
+            <th scope="col">Id</th>
+            <th scope="col">Title</th>
+            <th scope="col">Name</th>
+            <th scope="col">Surname</th>
+            <th scope="col">email</th>
+            <th scope="col">RoomId</th>
+            <th scope="col">Check in date</th>
+            <th scope="col">Check Out Date</th>
+            <th scope="col">Number Of Nights</th>
+          </tr>
+        </thead>
+        {props.results.map(customer => {
+          return (
+            <tbody>
+              <tr
+                className={isSelected ? "yellow-background" : null}
```

Comment:
Well done for getting the conditional rending of the class, it selects and deselects as it should. However, it does highlight and un-highlight every row when any of the rows are clicked, where could we apply this functionality so that only one row is selected when it is clicked, instead of all of them?

Reply:
I tried to work it out but really struggled with it.Thank you